dipy/tracking/local/localtracking.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+import random
```

Comment:
If we want to be really "strict", this import should be first in its own block since this is a core Python import.